### PR TITLE
End-to-end pixels

### PR DESCRIPTION
**Pull request by michaelfig**
The only remaining problem is "historical inaccuracy" when restarting a democlient that was following the chain.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -8,9 +8,9 @@ require (
 	github.com/bgentry/speakeasy v0.1.0
 	github.com/btcsuite/btcd v0.0.0-20190115013929-ed77733ec07d
 	github.com/btcsuite/btcutil v0.0.0-20180706230648-ab6388e0c60a
-	github.com/cosmos/cosmos-sdk v0.35.0
+	github.com/cosmos/cosmos-sdk v0.33.0
```
Comment (katelynsills) [inline go.mod]: hmm, is there a reason to go back to a previous version?

Comment (michaelfig) [reply go.mod]: When I was debugging the provisioning server, I found that 0.35.0 gave me problems.  I reverted for now until further testing can be done.  I need the testnet to be stable before I'm willing to try the newer version again.

File: lib/ag-solo/vats/repl.js [hunk 2]
```diff
@@ -112,12 +112,11 @@ export function addReplHandler(handler, E, homeObjects, sendBroadcast) {
           },
           rej => {
             // leave history[] alone: leave the rejected promise in place
-            display[histnum] = `Promise.reject(${stringify(rej)})`;
-          })
-        .then(_ => updateHistorySlot(histnum));
+            display[histnum] = `Promise.reject(${stringify(`${rej}`)})`;
```
Comment (warner) [inline lib/ag-solo/vats/repl.js]: I imagine my original stringify(error) wasn't doing much good, but maybe we should simplify this to just:

```js
`Promise.reject(${rej})`
```

I haven't experimented with it (I'm assuming that the template literal will apply the same `toString()` whether we do `${rej}` or something with nested literals), but doing a JSON.stringify of a string doesn't sound too productive :).

Comment (michaelfig) [reply lib/ag-solo/vats/repl.js]: It's done that way so that the resulting display is a legitimate Javascript expression.

File: go.mod (hunk 1 above)
Comment (michaelfig) [reply go.mod]: I just tried again with my "mfig" testnet, and indeed, upgrading to v0.35.0 causes the provisioning to hang forever.  So, I'm leaving this for now, but will create an issue to track.

**Pull request merged**